src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -37,12 +37,9 @@
 public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
 
     /*******************************************************************************************************
-    * @description the current Contact Settings object
-    */ 
-    private static Hierarchy_Settings__c ContactsSettings;
-    
-    private static ID householdRecTypeID = Schema.Sobjecttype.Account.getRecordTypeInfosByName().get('Household Account').getRecordTypeId();
-    private static ID adminRecTypeID = Schema.Sobjecttype.Account.getRecordTypeInfosByName().get('Administrative').getRecordTypeId();
+    * @description The selected Account record type.
+    */     
+    private static ID defaultRecTypeID = UTIL_CustomSettingsFacade.getSettings().Account_Processor__c;
```

Comment:
The getSettings method in the Façade always tries to get the user/profile ones first: https://github.com/SalesforceFoundation/HEDAP/blob/feature/216-acc-rec-types-change/src/classes/UTIL_CustomSettingsFacade.cls#L64